packages/services/api/src/modules/support/providers/support-manager.ts

```diff
@@ -568,6 +578,12 @@ export class SupportManager {
     });
     const customerType = this.resolveCustomerType(organization);
 
+    const formattedBody = ` "Category: " + ${request.data.category ? request.data.category : null}\n\n
+        "Project: " + ${request.data.project ? request.data.project : null}\n\n
+        "Target: " + ${request.data.target ? request.data.target : null}\n\n
```

Comment:
minor: since this is just text and not jsx, I'd prefer "not selected" as the default over `null`